src/lib/bootstrap_controller/bootstrap_controller.ml

```diff
@@ -273,6 +306,41 @@ end = struct
       let root_sync_ledger =
         Root_sync_ledger.create ledger_db ~logger:t.logger ~trust_system
       in
+      (let%bind () = Ivar.read ask_best_tip_signal in
+       Logger.info logger
+         "Requesting peers for their best tip to eargly start bootstrap"
+         ~module_:__MODULE__ ~location:__LOC__ ;
+       match%bind
+         download_best_tip ~network ~logger ~trust_system ~verifier
+           initial_root_transition
+       with
+       | Ok
+           ( ((peer_best_tip, _) as peer_best_tip_with_validation)
+           , peer_root_with_validation
+           , peer ) ->
+           let best_tip_candidate_state =
+             External_transition.consensus_state peer_best_tip.data
+           in
+           if not @@ worth_getting_root t best_tip_candidate_state then
+             Deferred.unit
+           else (
+             Logger.info logger
+               "Syncing with peer's best tip after asking other peers"
+               ~module_:__MODULE__ ~location:__LOC__
+               ~metadata:
+                 [ ("peer", Network_peer.Peer.to_yojson peer)
+                 ; ( "best tip"
+                   , External_transition.to_yojson peer_best_tip.data ) ] ;
+             Deferred.ignore
+               (sync_with_peer ~sender:peer.host ~root_sync_ledger t
```

Comment:
It concerns me that you are ignoring the result of this function call. Can you add a comment in the code explaining why it is ok to ignore that result? It seems to be an status indicator that seems worth checking.